### ISSUE TITLE
"Disable service" has no effect

### ISSUE THREAD

**Issue by logological** [meetfranz/franz#1406]
Right-clicking on a service brings up a context menu that includes a command called "Disable service".  This command has no apparent effect.

### Expected Behavior

I'm not sure exactly what this command is supposed to do.  Perhaps it should sign me out of the service and/or remove the service altogether?  Certainly it should do _something_, or else it needs to be removed.

### Current Behavior

Invoking the command has no apparent effect.

### Steps to Reproduce (for bugs)

1. Right-click on a service
2. Select "Disable service"

### Your Environment

* Franz Version used: 5.0.1
* Operating System and version: openSUSE 15.0 (x86-64)

**Comment by adlk** [comment]
I don't have openSUSE at hand but I can't reproduce this on other OSes. 

Just to make sure, 
- If you select the service in the tab bar, is site still active or does it show an "service is disabled" screen?
- Is the icon grayed out?

**Comment by logological** [comment]
I've tested again just now and I'm not able to reproduce the problem.  I right-clicked on a service's icon in the tab bar (whether or not I had first selected it by left-clicking on it) and selected "Disable service".  The icon then greys out and the service is disabled.

AFAIK I didn't do anything differently this time around.  Perhaps the only difference is that, when I was experiencing the bug, it was the first time I had run Franz on this machine.  I've since restarted the application.
